config/drupal-8/drupal-8-all-deprecations.php

```diff
@@ -11,7 +11,6 @@
 
     $containerConfigurator->import(PHPUnitSetList::PHPUNIT_60);
     $containerConfigurator->import(PHPUnitSetList::PHPUNIT_70);
-    $containerConfigurator->import(PHPUnitSetList::PHPUNIT_75);
```

Comment:
What is the replacement? We need this as it fixes PHPUnit code for folks.

Reply:
Answering for myself:

There is no replacement value. Removed in rector/rector-phpunit:0.11.26 due to it being a complex upgrade path.